src/theme/index.hbs

```diff
@@ -40,7 +40,7 @@
 
         {{#if mathjax_support}}
         <!-- MathJax -->
-        <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+        <script async type="text/javascript" src="{{ path_to_root }}mathjax.js"></script>
```

Comment:
I do mis the _?config=TeX-AMS-MML_HTMLorMML_ suffix.

So I'm asking  should it be
```
<script async type="text/javascript" src="{{ path_to_root }}mathjax.js?config=TeX-AMS-MML_HTMLorMML">
</script>
```
?